src/screens/signup/PhoneNumberScreen.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+import { Text, View, TextInput, TouchableOpacity } from 'react-native';
+import PhoneInput from 'react-native-phone-input';
+import SelectBox from 'react-native-multi-selectbox';
+import { ThemeProvider } from 'styled-components';
+import styles from '../../styles/allStyles';
+
+const { allStyles, ourColors } = styles;
+/**
+ * @class
+ */
+class PhoneNumberScreen extends Component {
+  /**
+* @constructor
+*/
+  constructor() {
+    super();
+    this.state = {
+      countryCodes: [],
+      countryNames: [],
+      selectedCode: [],
+    };
+  }
+
+  /**
+   * @method
+   * @returns {*} state
+   * @description it sets the state
+   */
+  componentDidMount = () => {
+    const allCountries = this.phone.getPickerData();
+    const countryCodes = allCountries.map((currItem) => ({
+      item: currItem.dialCode, id: (currItem.dialCode),
+    }));
+    const countryNames = allCountries.map((currItem) => ({
+      countryNames: currItem.label,
+    }));
+    this.setState({ countryCodes, countryNames });
+  }
+
+  /**
+   * @method
+   * @return {*} next state
+   */
+  nextFunction = () => {
+    // console.log(this.state);
+  }
+
+  /**
+* @method
+* @return {*} UI
+* @description it returns the UI
+*/
+  render() {
+    const { countryCodes, countryNames, selectedCode } = this.state;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['countryNames' is assigned a value but never used.](https://app.codacy.com/gh/nezago/phone-history-store-frontend/pullRequest?prid=5447061)